nodeletctl/pkg/nodeletctl/deployer.go

```diff
@@ -39,20 +41,72 @@ func NewNodeletDeployer(cfg *BootstrapConfig, sshClient ssh.Client,
 	return deployer
 }
 
-func GetNodeletDeployer(cfg *BootstrapConfig, clusterStatus *ClusterStatus, nodeletCfg *NodeletConfig, nodeName, nodeletSrcFile string, sshKey []byte) (*NodeletDeployer, error) {
-	sshClient, err := CreateSSHClient(nodeName, cfg.SSHUser, sshKey, 22)
+func GetNodeletDeployer(cfg *BootstrapConfig, clusterStatus *ClusterStatus, nodeletCfg *NodeletConfig, nodeName, nodeletSrcFile string, sshKeyFile string) (*NodeletDeployer, error) {
```

Comment:
@roopakparikh we don't need to pass in the SSH key file path, it is part of the BootstrapConfig struct - cfg.SSHKeyPrivateFile

Reply:
checking